app/uk/gov/hmrc/agentsubscription/model/subscriptionJourney/ContactTelephoneData.scala

```diff
@@ -16,11 +16,39 @@
 
 package uk.gov.hmrc.agentsubscription.model.subscriptionJourney
 
-import play.api.libs.json.{Format, Json}
+import play.api.libs.json.{Format, JsResult, JsValue, Json, Writes}
+import uk.gov.hmrc.agentsubscription.repository.EncryptionUtils.decryptOptString
+import uk.gov.hmrc.crypto.json.JsonEncryption.stringEncrypter
+import uk.gov.hmrc.crypto.{Decrypter, Encrypter}
 
-case class ContactTelephoneData(useBusinessTelephone: Boolean, telephoneNumber: Option[String])
+case class ContactTelephoneData(
+  useBusinessTelephone: Boolean,
+  telephoneNumber: Option[String],
+  encrypted: Option[Boolean] = None
+)
 
 object ContactTelephoneData {
+  def databaseFormat(implicit crypto: Encrypter with Decrypter): Format[ContactTelephoneData] = {
 
-  implicit val format: Format[ContactTelephoneData] = Json.format[ContactTelephoneData]
+    def reads(json: JsValue): JsResult[ContactTelephoneData] =
+      for {
+        isEncrypted <- (json \ "encrypted").validateOpt[Boolean]
+        result = ContactTelephoneData(
+                   (json \ "useBusinessTelephone").as[Boolean],
+                   decryptOptString("telephoneNumber", isEncrypted, json),
+                   (json \ "encrypted").asOpt[Boolean]
+                 )
+      } yield result
```

Comment:
Should probably yield the ContactTelephoneData as the other examples have done for a consistent code style

Reply:
done